config/env/development.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
     }
   },
   app: {
-    title: defaultEnvConfig.app.title + ' - Development Environment'
+    title: defaultEnvConfig.app.title + ' - Development Environment',
+    domain: process.env.DOMAIN || 'localhost:3000'
```

Comment:
you can do here:
```javascript
domain: process.env.DOMAIN || 'localhost:' + defaultEnvConfig.port;
```
I think it's better because this way you don't hard code the port number.